.docker/entrypoint/lib/commands/verifier.rb

```diff
@@ -40,17 +34,42 @@ def verify(ignore_urls = nil)
         private
 
         def options(ignore_urls = nil)
-          opts = {
+          ignore_urls = massage(ignore_urls)
+
+          opts = default_options
+          opts[:log_level] = @log_level.to_sym unless @log_level.nil?
+          opts[:url_ignore] = ignore_urls if ignore_urls.valid_array?
+
+          opts
+        end
+
+        def default_options
+          {
             assume_extension: true,
             check_html: true,
             enforce_https: true,
             only_4xx: true,
             check_unrendered_link: true
           }
+        end
 
-          return opts if ignore_urls.nil? || !ignore_urls.is_a?(Array) || ignore_urls.empty?
+        def massage(ignore_urls)
+          return [ignore_urls] if ignore_urls.is_a?(Regexp) || ignore_urls.is_a?(String)
```

Comment:
Codacy found an issue: [Modifier form of `if` makes the line too long.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)